python/feathub/processors/flink/table_builder/sliding_window_utils.py

```diff
@@ -108,16 +112,22 @@ def evaluate_sliding_window_transform(
     flink_table: NativeFlinkTable,
     window_descriptor: SlidingWindowDescriptor,
     agg_descriptors: List["AggregationFieldDescriptor"],
+    empty_window_output_enabled: bool,
+    deduplicate_output_enabled: bool,
 ) -> NativeFlinkTable:
     """
     Evaluate the sliding window transforms on the given flink table and return the
     result table.
 
+
     :param t_env: The StreamTableEnvironment of the `flink_table`.
     :param flink_table: The input Flink table.
     :param window_descriptor: The descriptor of the sliding window.
     :param agg_descriptors: A list of descriptor that descriptor the aggregation to
                             perform.
+    :param deduplicate_output_enabled: Whether to deduplicate sliding window output.
```

Comment:
FeatHub users typically will not read this Python doc.

Should we document the semantics of these two configs in `sliding_window_transform.py`?

And should we add a TODO regarding retracting the value when the window becomes empty?